### PR TITLE
adding nbcelltests

### PR DESCRIPTION
**Pull request by timkpaine**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by timkpaine** [comment]
https://github.com/conda-forge/staged-recipes/pull/11431

**Comment by timkpaine** [comment]
@ceball @vidartf please confirm

**Comment by ceball** [comment]
Thanks for doing this. I'm willing to be listed as a maintainer.

**Comment by vidartf** [comment]
🎉

**Comment by timkpaine** [comment]
@conda-forge/help-python all set!

File: recipes/nbcelltests/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - nodejs
+  run:
+    - python >=3.7
+    - nbval >=0.9.1
+    - nbconvert
+    - pytest >=4.4.0
+    - pytest-cov
+    - pytest-html >=1.20.0
+    - flake8
+
```
Comment (nehaljwani) [inline recipes/nbcelltests/meta.yaml:31]: The wheel on pypi also has:
```
Requires-Dist: jupyterlab (>=1.0.0)
```
Could you confirm that it is not necessary?

**Comment by nehaljwani** [comment]
@conda-forge-admin restart ci

File: recipes/nbcelltests/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 904656317adb86faba5cb5640195e79ea5022eab2ab2852a53975b5346f3659d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
```
Comment (nehaljwani) [inline recipes/nbcelltests/meta.yaml:19]: The wheel on pypi also work with py36. Was the minimum constraint on 3.7 intentional here (and below)?

Comment (ceball) [inline recipes/nbcelltests/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/nbcelltests/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - pip
+    - jupyter-packaging
+    - nodejs
+  run:
+    - python >=3.7
```
Comment (ceball) [inline recipes/nbcelltests/meta.yaml:24]:
```suggestion
    - python >=3.6
```

**Comment by ceball** [comment]
Thanks for reviewing, Nehal!

**Comment by nehaljwani** [comment]
Thank you @ceball!

**Comment by nehaljwani** [comment]
Thank you @timkpaine !

**Pull request merged**